tests/testthat/test-build-article.R

```diff
@@ -276,6 +276,13 @@ test_that("warns about missing images", {
   expect_snapshot(build_article("kitten", pkg))
 })
 
+test_that("spaces in base paths do work", {
+  d <- withr::local_tempdir("basename with spaces")
+  fs::dir_copy(test_path("assets/articles-images"), d, overwrite = TRUE)
```

Comment:
Could you please use `local_pkgdown_site()` and `pkg_add_file()` as in the next test down?

Reply:
Done: cc095ae2